syntax = "proto3";

import public "proto/data.proto";

message ScheduleResponse {
    // Maybe repeated repeated Course?
    // repeated Course fall1   = 1;
    // repeated Course spring1 = 2;
    // repeated Course fall2   = 3;
    // repeated Course spring2 = 4;
    // repeated Course fall3   = 5;
    // repeated Course spring3 = 6;
    // repeated Course fall4   = 7;
    // repeated Course spring4 = 8;
    map<string, CourseList> course_map = 1;
}


message ProfessorResponse {
    repeated Professor professors = 1;
}

message CourseResponse {
    repeated Course courses  = 1;
}

message DebugResponse {
    string msg = 1;
}

message MajorResponse {
    repeated Major majors = 1;
}

message NotificationResponse {
    bool success = 1;
}